test/e2e/node/pod_resize.go

```diff
@@ -348,13 +348,81 @@ func doPodResizeSchedulerTests(f *framework.Framework) {
 		framework.Logf("TEST2: Pod '%s' CPU requests '%dm'", testPod3.Name, testPod3.Spec.Containers[0].Resources.Requests.Cpu().MilliValue())
 		framework.ExpectNoError(e2epod.WaitForPodRunningInNamespace(ctx, f.ClientSet, testPod3))
 
-		ginkgo.By("deleting pods")
-		delErr1 := e2epod.DeletePodWithWait(ctx, f.ClientSet, testPod1)
-		framework.ExpectNoError(delErr1, "failed to delete pod %s", testPod1.Name)
+		// Scheduler focussed pod resize E2E test case #3
```

Comment:
the two new tests aren't really "scheduler focused" ig but it was convenient to build on these existing tests to verify the deferred/infeasible conditions